pkg/k8s/synced/crd.go

```diff
@@ -163,34 +158,34 @@ func (k *K8sWatcher) crdsInit(ctx context.Context, watcherClient apiextclientset
 	}
 }
 
-func addCRD(crds *crdState, obj interface{}) {
+func (s *crdState) addCRD(obj interface{}) {
```

Comment:
I think we can rename these now to simply `add` and just drop the `CRD` part completely as it's redundant.

Reply:
Done, changed "deleteCRD" to "remove" as well.